SourceCode/include/triangle.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include "vec.hpp"
+
+class Triangle
+{
+  Triangle() = default;
+  Triangle(const cm::Vec3& v0, const cm::Vec3& v1, const cm::Vec3& v2);
+
+  cm::Vec3 v0;
+  cm::Vec3 v1;
+  cm::Vec3 v2;
```

Comment:
It's often very useful to be able to access the vertices using indexing, so an array of 3 elements is very helpful.